system-int-test/src/cmr/system_int_test/utils/service_util.clj

```diff
@@ -224,3 +227,77 @@
     (is (= 200 status))
     (is (= (set (map :concept-id expected-colls))
            (set colls)))))
+
+(defn- assert-collection-atom-result
+  "Verify the collection ATOM response has-formats, has-variables, has transforms fields
+  have the correct values"
+  [coll expected-fields]
+  (let [coll-with-extra-fields (merge coll expected-fields)
+        {:keys [entry-title]} coll
+        coll-atom (atom/collections->expected-atom
+                   [coll-with-extra-fields]
+                   (format "collections.atom?entry_title=%s" entry-title))
+        {:keys [status results]} (search/find-concepts-atom
+                                  :collection {:entry-title entry-title})]
+
+    (is (= [200 coll-atom]
+           [status results]))))
+
+(defn- assert-collection-json-result
+  "Verify the collection JSON response associations related fields have the correct values"
+  [coll expected-fields serv-concept-ids var-concept-ids]
+  (let [coll-with-extra-fields (merge coll
+                                      expected-fields
+                                      {:services serv-concept-ids
+                                       :variables var-concept-ids})
```

Comment:
Just trying to learn sth here...  why is that tags don't need to be here?

Reply:
We have issue written to move tags in here as well. We have not done that because tags are being actively used by other systems (e.g EDSC), and we have to coordinate with those systems to make the change.